src/Type/CommonType.ts

```diff
@@ -43,5 +43,20 @@ export interface FormFieldsProps {
   type?: string
 }
 
-
 export type NullableDateArray = (Date | null)[] | null
+
+export interface Role {
+  name: string
+}
+
+export interface Country {
+  id: number
+  name: string
+  label: undefined
```

Comment:
will be only undefined ?
then why you need it ?

Reply:
actually getting null in response , that's why making it undefined for now